src/index.js

```diff
@@ -0,0 +1,8 @@
+import Server from './Server';
+import config from './config/configuration';
+import schema from './modules';
+
+const server = new Server(config);
+(() => {
+  server.bootstrap().setupApollo(schema);
+})();
```

Comment:
Any particular reason to create an immediate function call

Reply:
I thought this function will not run without, arrow function, but its working fine when i removed arrow function. I have updated the code.